cartography/data/jobs/analysis/aws_lambda_function_asset_exposure.json

```diff
@@ -0,0 +1,18 @@
+{
+	"statements": [
+		{
+			"query": "MATCH (lf:AWSLambda)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) where EXISTS(lf.exposed_internet) WITH lf LIMIT $LIMIT_SIZE REMOVE lf.exposed_internet, lf.exposed_internet_type return COUNT(*) as TotalCompleted",
+			"iterative": true,
+			"iterationsize": 1000
+		},
+		{
+			"query": "MATCH (lf:AWSLambda{public_policy: true})<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WITH lf WHERE (NOT EXISTS(lf.exposed_internet_type)) OR (NOT 'public_policy' IN lf.exposed_internet_type)\nSET lf.exposed_internet = true, lf.exposed_internet_type = coalesce(lf.exposed_internet_type , []) + 'public_policy';",
+			"iterative": false
+		},
+		{
+			"query": "MATCH (lf:AWSLambda)<-[:RESOURCE]-(:AWSAccount{id: $AWS_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WITH lf WHERE ((NOT EXISTS(lf.vpc_id)) OR (lf.vpc_id = '')) AND (NOT EXISTS(lf.exposed_internet_type)) OR (NOT 'vpc_none' IN lf.exposed_internet_type)\nSET lf.exposed_internet = true, lf.exposed_internet_type = coalesce(lf.exposed_internet_type , []) + 'vpc_none';",
```

Comment:
this rule not in public facing document can you share the documentation for this rule?

Reply:
That rule was implemented in get_lambda_data function.
![image](https://user-images.githubusercontent.com/42804494/223197891-7a7f2a07-b6ef-4f1d-85e9-a919d074203b.png)